.evergreen/config.yml

```diff
@@ -2507,6 +2510,13 @@ axes:
         variables:
           test_encryption: true
         batchtime: 10080  # 7 days
+      - id: "encryption_pyopenssl"
+        display_name: "Encryption PyOpenSSL"
+        tags: ["encryption_tag"]
+        variables:
+          test_encryption: true
+          test_encryption_pyopenssl: true
+        batchtime: 10080  # 7 days
```

Comment:
What if we only test 1 server version instead of the whole matrix here? We could use the `rules: if/then/add_tasks/remove_tasks` feature like we do here: https://github.com/mongodb/mongo-python-driver/blob/cbad35ec9d591b09c492f4c869c50fd203b55e44/.evergreen/config.yml#L2849

Reply:
Fixed, we now test `latest-replica-set` with this variant and all server types otherwise.